src/utils/order.ts

```diff
@@ -1,6 +1,6 @@
 import { OrderType, REVERSE_REACTOR_MAPPING } from "../constants";
 import { MissingConfiguration } from "../errors";
-import { DutchOrder, Order } from "../order";
+import { DutchOrder, Order, RelayOrder } from "../order";
```

Comment:
Is the reason why `RelayOrder` can't implement `Order` that it doesn't have validation related stuff? Can probably make it work by moving `validate()` into DutchOrder but I don't have a strong opinion here

Reply:
Yea mostly just because it doesn't have validation logic. That's a good idea, I've pushed up commits where I have the base Order class generic and it feels a bit cleaner